components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -1580,8 +1580,9 @@ def load_metadata_preview(request, c_type, c_id, conn=None, share_id=None,
             act = "-"
             if c['active']:
                 act = ""
-            chs.append('%s%s|%d:%d$%s'
-                       % (act, i+1, c['start'], c['end'], c['color']))
+            reverse = 'r' if c['reverseIntensity'] else '-r'
+            chs.append('%s%s|%d:%d%s$%s'
```

Comment:
if you went with the proposal for ':' separation then it'd need to be

`chs.append('%s%s|%d:%d:%s$%s'`